_overviews/FAQ/index.md

```diff
@@ -108,20 +108,25 @@ Your sbt 1.x build definition is always a Scala 2.12 program.
 Regardless, in your `build.sbt`, you can set `scalaVersion` to whichever
 available distribution you want and your program code will be compiled with that version.
 
-### I want Scala 3.1.1 (etc); why does std lib say it's using Scala 2.13?
+### I want Scala 3. Why does `versionNumberString` say I'm on 2.13?
 
-Scala 3 currently uses the Scala 2.13 library by leveraging its seamless
-interoperability. Note that it does not necessarily ingest the latest
-version of the Scala 2.13 library.
+To aid migration, Scala 3 currently uses the Scala 2.13 library as-is.
+That's why `versionString` and `versionNumberString` give a
+perhaps-unexpected Scala 2 answer:
```

Comment:
I wonder what would be the German word for "a perhaps-unexpected Scala 2 answer".

"That's why [they] report a Scala 2 version."